examples/platforms/da15000/radio.c

```diff
@@ -31,528 +31,344 @@
 * Platform abstraction for radio communication.
 */
 
-#include <fcntl.h>
-#include <string.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <unistd.h>
 #include <common/code_utils.hpp>
 #include <platform/alarm.h>
 #include <platform/radio.h>
-#include <openthread.h>
 #include "platform-da15000.h"
 
-#define DEBUG_LOG_ENABLE (0)
-
-#include <ftdf.h>
 #include <ad_ftdf.h>
-#include "ad_ftdf_phy_api.h"
-#include "hw_rf.h"
-#include <global_io.h>
+#include <ad_ftdf_phy_api.h>
+#include <hw_rf.h>
+#include <internal.h>
 #include <regmap.h>
-#include "black_orca.h"
-#include "core_cm0.h"
-#include "internal.h"
 #include <hw_uart.h>
 
-typedef struct __attribute__((packed)) RadioMessage
-{
-    uint8_t mChannel;
-    uint8_t mPsdu[kMaxPHYPacketSize];
-} RadioMessage;
-
-static PhyState s_state = kStateDisabled;
-static RadioMessage s_receive_message;
-static RadioMessage s_transmit_message;
-static RadioMessage s_ack_message;
-static RadioPacket s_receive_frame;
-static RadioPacket s_transmit_frame;
-static RadioPacket s_ack_frame;
-static bool s_data_pending = false;
-static bool SendFrameDone = false;
-static uint8_t s_extended_address[OT_EXT_ADDRESS_SIZE];
-static uint16_t s_short_address;
-static uint16_t s_panid;
-
-//static int s_sockfd;
-
-static bool s_promiscuous = false;
-
-static uint8_t sleep_init_delay = 0;
-static bool SED = false;
-static uint32_t sleep_const_delay;
-
-static otInstance *First_Instace;
-uint32_t NODE_ID = 1;
-
 #define DEFAULT_CHANNEL           (11)
-#define FTDF_OFFSET_CHANNEL       (11)
 
-void disable_interrupt()
-{
-    NVIC_DisableIRQ(FTDF_GEN_IRQn);
-}
+static otInstance *ThreadInstance;
```

Comment:
style: statically-declared variable names should be prefixed with `s` (e.g. `sThreadInstance` in this case).  This applies to the statically-declared variables names below as well.